pkg/app/app_reconcile_test.go

```diff
@@ -223,3 +225,16 @@ func Test_TemplateError_DisplayedInStatus_UsefulErrorMessageProperty(t *testing.
 		reflect.DeepEqual(expectedStatus, crdApp.app.Status()),
 		fmt.Sprintf("Status is not same:\nExpected:\n%#v\nGot:\n%#v\n", expectedStatus, crdApp.app.Status()))
 }
+
+type FakeComponentInfo struct {
```

Comment:
For now this is kinda duplicated across the tests.... Probably a better centralized place for it...